pkg/platform/model/buildplanner/build.go

```diff
@@ -118,6 +119,11 @@ func processBuildPlannerError(bpErr error, fallbackMessage string) error {
 		if ok && code == clientDeprecationErrorKey {
 			return &response.BuildPlannerError{Err: locale.NewExternalError("err_buildplanner_deprecated", "Encountered deprecation error: {{.V0}}", graphqlErr.Message)}
 		}
+		if status, ok := graphqlErr.Extensions[statusExtensionKey].(float64); ok {
+			if status >= 400 && status < 500 {
+				return locale.NewInputError("err_buildplanner_api_input_error", "{{.V0}}: {{.V1}}", fallbackMessage, bpErr.Error())
+			}
+		}
 	}
 	return &response.BuildPlannerError{Err: locale.NewExternalError("err_buildplanner", "{{.V0}}: Encountered unexpected error: {{.V1}}", fallbackMessage, bpErr.Error())}
```

Comment:
Wait so we were reporting this? It seems like external error is bugged then.

None of these changes are necessary I think. This sounds like a bug in reporting of external errors. External errors should still be shown to the user, external just means that we don't report it to rollbar. I think the fix here should be in our error printing of external errors.

Reply:
Ugh yeah, it was bugged :(